### PR TITLE
New package: phpMyAdmin-4.8.5

### PR DESCRIPTION
**Pull request by Francesco149**
config file can't be chmodded to 600 otherwise it complains it can't access it, other than that, I tested with lighttpd and mysqld and it works fine

### CODE REVIEW

File: srcpkgs/phpMyAdmin/template [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+short_desc="Web interface for MySQL and MariaDB"
+maintainer="Franc[e]sco <lolisamurai@tfwno.gf>"
+license="GPL-2.0-or-later"
+homepage="https://www.phpmyadmin.net"
+distfiles="https://files.phpmyadmin.net/phpMyAdmin/${version}/phpMyAdmin-${version}-all-languages.tar.xz"
+checksum=d7ab5df4e464d7ba17bf2a42da7d7f26dad45c34bf321ac8ae7d2ed748413913
+
+do_install() {
+	vmkdir usr/share/webapps/${pkgname}
+	cp -rT ${wrksrc} ${DESTDIR}/usr/share/webapps/${pkgname}
```
Comment (maxice8) [inline srcpkgs/phpMyAdmin/template]: i think vcopy * can be used, no ?

Comment (Francesco149) [reply srcpkgs/phpMyAdmin/template]: oh yeah, i copypasta'd too much from another template

**Comment by Chocimier** [comment]
Please replace all ${pkgname} with phpMyAdmin

**Comment by Francesco149** [comment]
also removed LICENSE since it's just gpl

**Pull request merged**